environment/src/test/java/com/sequenceiq/environment/environment/v1/converter/FreeIpaConverterTest.java

```diff
@@ -136,9 +146,7 @@ public void testConvertWithImage() {
         AttachedFreeIpaRequest request = new AttachedFreeIpaRequest();
         request.setCreate(true);
         request.setImage(aFreeIpaImage(IMAGE_CATALOG, IMAGE_ID));
-        request.setEnableMultiAz(true);
```

Comment:
why it is removed?

Reply:
This is to make it clear that we are not testing for multiAz here. I have added separate tests for multiAz. Please refer to added tests.